### DISCUSSION TITLE
개별 사용자의 음량을 조절할 수 있으면 좋겠습니다.

### DISCUSSION THREAD

**Discussion by Gukmin** [metamong-friends/metamong-friends.github.io#38] [제안하기]
사용자마다 마이크 음량이 달라, 사운드가 너무 작거나 큰 경우가 종종 발생합니다. 
현재 개인 마이크 음량 조절은 가능하지만 개별 사용자의 음량 조절은 컨트롤 할 수 없어 보이는데,  도입되면 더욱 편하게 사용 할 수 있을 것 같습니다!

**Comment by ACT-TJ** [comment]
안녕하세요!!

있다면 편리한 기능일 것 같습니다.

저희도 벤치마킹 통해 여러 유사 서비스에 들어있는 것을 확인하였습니다.^^
개선 기능으로 분류하여 관리할 수 있도록 하겠습니다. 
음성대화 관련해서는 기본기능에 부족함을 좀 더 느끼시는 상황이라 편하게 대화가 되는 상황을
우선 최대한 만든 뒤 기능을 하나씩 추가해 갈 예정입니다. ( 따라서  개선에 시간이 걸릴 수 있습니다^^;; )

감사합니다.

**Comment by shlee303** [comment]
@Gukmin 해당 기능 릴리즈 됐습니다 :) 좋은 의견 감사합니다~!!!
